app/services/open_library_service.rb

```diff
@@ -2,17 +2,18 @@
 
 class OpenLibraryService
   include HTTParty
-
+  base_uri 'https://openlibrary.org'
   def initialize(params)
     @isbn = params[:isbn]
   end
 
   def book_info
-    info = HTTParty.get('https://openlibrary.org/api/books?bibkeys=ISBN:' + @isbn +
-                        '&format=json&jscmd=data')
-    info = info['ISBN:' + @isbn]
-    book = info.slice(:title, :subtitle, :number_of_pages, :authors)
-    book[:isbn] = @isbn
-    book
+    response = OpenLibraryService.get('/api/books', query: { bibkeys: 'ISBN:' + @isbn,
+                                                             format: 'json', jscmd: 'data' })
+    if response.nil?
+      response
```

Comment:
Do you really want to ignore errors? It's not really a correction since each project will handle this differently (sometimes a request is really important and you want the request to throw an error, sometimes you want to silently ignore it), just remember that it's a choice you're actively making